dts/bindings/pwm/ite,it8xxx2-pwm.yaml

```diff
@@ -28,6 +28,16 @@ properties:
       3 = PWM_CHANNEL_3, 4 = PWM_CHANNEL_4, 5 = PWM_CHANNEL_5,
       6 = PWM_CHANNEL_6, 7 = PWM_CHANNEL_7
 
+  init-output-level:
+    type: int
+    enum:
+      - 0
+      - 1
+      - 255
+    description: |
+      0 = PWM_IT8XXX2_INIT_OUTPUT_LOW, 1 = PWM_IT8XXX2_INIT_OUTPUT_HIGH,
+      255 = PWM_IT8XXX2_INIT_OUTPUT_NONE
```

Comment:
hey, something similar came up in the past in the context of display backlight control, I'm thinking it would make sense to have this property in `pwm-controller.yaml` and the definitions in `include/zephyr/dt-bindings/pwm/pwm.h`, what do you guys think? @anangl @henrikbrixandersen